samples/attested_tls/server/enc/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ add_custom_command(OUTPUT tls_server_enc.signed tls_server_enc_mrenclave.h
   COMMAND openenclave::oesign sign -e $<TARGET_FILE:tls_server_enc> -c ${CMAKE_SOURCE_DIR}/server/enc/enc.conf -k ${CMAKE_SOURCE_DIR}/server/enc/private.pem
   COMMAND openenclave::oesign dump -e tls_server_enc.signed  > temp.dmp
   COMMAND bash ${CMAKE_SOURCE_DIR}/scripts/gen_mrenclave_header.sh ${CMAKE_SOURCE_DIR}/common/tls_server_enc_mrenclave.h temp.dmp
+  COMMAND ${CMAKE_COMMAND} -E sleep 1
```

Comment:
Does sleeping here ensure that the bash process from the previous line completes?

Reply:
yes